src/silverlabnwb/nwb_file.py

```diff
@@ -583,15 +586,37 @@ def add_stimulus(self):
             self.nwb_file.add_stimulus(TimeSeries(**attrs))
         self._write()
 
-    def read_cycle_relative_times(self, file_path):
+    def read_cycle_relative_times(self, folder_path):
         """Read the 'Single cycle relative times.txt' file and store the values in memory.
```

Comment:
See the [previous comment](https://github.com/SilverLabUCL/PySilverLabNWB/pull/58#discussion_r456367124) about this.